src/controllers/expense.controller.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const expenseService = require('../services/expense.service');
+const { getAll } = require('../services/user.service');
+
+const get = (req, res) => {
```

Comment:
There is a lot of bussiness logic happening in the controller itself. It should be extracted to the related service.